app/api/score_sub.py

```diff
@@ -273,10 +273,11 @@ async def submit_score(
 
         # send request to rmq
         if app.state.services.amqp_channel is not None:
-            await app.state.services.amqp_channel.default_exchange.publish(
-                aio_pika.Message(body=orjson.dumps(submission_request)),
-                routing_key="score_submission",
-            )
+            for routing_key in config.SCORE_SUBMISSION_ROUTING_KEYS:
+                await app.state.services.amqp_channel.default_exchange.publish(
+                    aio_pika.Message(body=orjson.dumps(submission_request)),
+                    routing_key=routing_key,
+                )
```

Comment:
not sure if i understand this - why would we want to send the same event data to multiple keys? could we instead have multiple consumers consume from the generic `score_submission` event?

Reply:
nvm i think i'm wrong ?

Reply:
ok i'm not

there are multiple consumers allowed butt amqp spec expects to load balance between them, so no 2 consumers will receive the same message

Reply:
damn, kafka's way of doing this is through "consumer groups"

Reply:
this might be how they represent it? not sure: https://blog.rabbitmq.com/posts/2022/07/rabbitmq-3-11-feature-preview-super-streams/